.shippable.yml

```diff
@@ -32,6 +32,14 @@ build:
         options: "-e HOME=/home/buildslave --privileged=true --tty --net=bridge --user buildslave"
 
     ci:
+      - sudo pip3 install west
```

Comment:
We already depend on many other python modules that are not fixed to a particular version. The latest revision of `west` is guaranteed to work with the latest master. If we need to pin a particular release to a version of `west` this can be done in the repository itself

Reply:
> that are not fixed to a particular version. 

In CI they are. You know that 2 CI runs using the same docker image will use the same version of that tool.
With this change, if somebody pushes to west's master a breaking change (say a bug), things will suddenly break here. That simply should not be a possibility in a proper CI environment. Here we are testing a Zephyr PR, therefore everything else should be constant.

Reply:
west is already in the CI docker image, so this can be removed.